pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotBrokerDebug.java

```diff
@@ -157,4 +163,8 @@ public Map<ServerInstance, List<String>> getRoutingTableForQuery(
   public String getServerRoutingStats() {
     return _serverRoutingStatsManager.getServerRoutingStatsStr();
   }
+
+  private int getRequestId() {
+    return (int) (_requestIdGenerator.getAndIncrement() % MAX_REQUEST_ID);
```

Comment:
I don't think we need to limit the max request id here. It is done in the `InstanceSelector`

Reply:
Ah good point. Done.